src/Prometheus.Client.MetricPusher.HostedService.Tests/Prometheus.Client.MetricPusher.HostedService.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CleanMoq" Version="1.0.0" />
+    <PackageReference Include="FluentAssertions" Version="6.11.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
```

Comment:
For each target framework specify `Microsoft.Extensions.DependencyInjection` version, like this:
```xml
  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'"> 
    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.32" />
  </ItemGroup>
  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
  </ItemGroup>
  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
  </ItemGroup>
```

Reply:
@phnx47 
Do we really need to do so since v7.0.0 has compatibility with all of em...
![Capture](https://github.com/prom-client-net/prom-client-metricpusher-hostedservice/assets/26391829/5995e597-5f75-4390-80b8-2575b980ff03)

Reply:
Yes, want to test that code still compatible with old `Microsoft.Extensions.DependencyInjection`. Many .NET devs don't update to latest version.